### ISSUE TITLE
Terraform Plan Crash

### ISSUE THREAD

**Issue by JulianGindi** [hashicorp/terraform#1059]
Crashlog [here](https://gist.github.com/JulianGindi/7b4a69064aa7be97046d)

Just ran into this when running `terraform plan` - not sure how to proceed.

**Comment by phinze** [comment]
Hey there, sorry about the trouble!

Here's the relevant part of the crash log:

```
2015/02/25 21:32:44 [INFO] Module root walking: aws_autoscaling_group.web-autoscale (Graph node: aws_autoscaling_group.web-autoscale)
2015/02/25 21:32:44 [DEBUG] aws_autoscaling_group.web-autoscale: Executing diff
2015/02/25 21:32:44 terraform-provider-aws: panic: Unknown kind: string
```

Can you provide the terraform config you were using that generated the crash and I can try to reproduce?

**Comment by mitchellh** [comment]
This was fixed in 0.3.7, or master... but either way its caused by weird types going into a parameter. To verify, can you show us your config?
